autogen/main/firewall.tf.tmpl

```diff
@@ -112,6 +112,7 @@ resource "google_compute_firewall" "master_webhooks" {
   direction   = "INGRESS"
 
   source_ranges = [local.cluster_endpoint_for_nodes]
+  source_tags   = []
```

Comment:
Workaround for  hashicorp/terraform-provider-google#10494. Long term fix maybe GoogleCloudPlatform/magic-modules#5526 but will need to validate.